A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1779,6 +1816,13 @@ var A320_Neo_UpperECAM;
             const percent = SimVar.GetSimVarValue(name, "percent");
             return percent;
         }
+        getDisplayActiveEngine(_engine) {
+            if (this.getValue(_engine) < 57.8 && this.getEngineStartStatus(_engine) && this.getIgnitionStatus(_engine)) {
```

Comment:
Simplane.getEngineActive(0) || Simplane.getEngineActive(1)
returns true if engine has N2 > 20% (could be less, not sure)

Reply:
Not sure what advantage that conveys over the proposed code change.